external-crates/move/crates/move-compiler/src/expansion/translate.rs

```diff
@@ -879,6 +879,11 @@ fn check_visibility_modifiers(
         for (_, _, friend_decl) in friends {
             let loc = friend_decl.loc;
             let diag = if edition == Edition::E2024_MIGRATION {
+                for aloc in &friend_decl.attr_locs {
+                    context
+                        .env()
+                        .add_diag(diag!(Migration::RemoveFriend, (*aloc, friend_msg)));
+                }
```

Comment:
Should we just grab the min here?

Reply:
My initial plan was to do that for the whole thing, so:

```
#[test_only]
friend a::m;
```

would become

```
/* #[test_only]
friend a::m; */
```

Unfortunatley this gets a little weird with comments and multiple attributes:

```
/* #[test_only]
#[ext(some_exit)]
// friend for some purpose
friend a::m; */
```

I thought this ultimately looked cleaner when reading the resultant migration code:

```
/* #[test_only] */
/* #[ext(some_exit)] */
// friend for some purpose
/* friend a::m; */
```